packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -66,21 +66,31 @@ export async function createNewFluidFile(
 
     let itemId: string;
     let summaryHandle: string = "";
-
+    let sharingLink: string = "";
+    let sharingLinkErrorReason: string = "";
     if (createNewSummary === undefined) {
         itemId = await createNewEmptyFluidFile(getStorageToken, newFileInfo, logger, epochTracker);
     } else {
         const content = await createNewFluidFileFromSummary(
             getStorageToken, newFileInfo, logger, createNewSummary, epochTracker);
         itemId = content.itemId;
         summaryHandle = content.id;
+        sharingLink = content.sharingLink || "";
+        sharingLinkErrorReason = content.sharingLinkErrorReason || "";
```

Comment:
Instead of putting empty strings, undefined is fine. That way you can remove the or statement and the initialization above.